### PR TITLE
[Teams] fixes to tunnel docs

### PR DESCRIPTION
**Pull request by abracchi-tw**

### CODE REVIEW

File: products/cloudflare-one/src/content/connections/connect-apps/configuration/configuration-file/ingress.md [hunk 1]
```diff
@@ -34,6 +34,12 @@ ingress:
 
 ## Matching traffic
 
+<Aside type='note'>
+
+You can use wildcards to match traffic to multiple subdomains or paths. For example, if you set the `hostname` key to `*.example.com`, both `test.example.com` and `try.example.com` will route traffic to your origin.
```
Comment (sudarshan-reddy) [inline products/cloudflare-one/src/content/connections/connect-apps/configuration/configuration-file/ingress.md:39]: I have a minor nit here: the hostname can be `example.com` without the asterisk and the rest of the statement still holds true.

File: products/cloudflare-one/src/content/faq/teams-troubleshooting.md [hunk 2]
```diff
@@ -71,6 +71,19 @@ This is due to a Google policy change requiring you to set your Google Admin con
...
+## I see an error: x509: certificate signed by unknown authority.
+
+This means the origin is using a certificate that `cloudflared` does not trust. For example, you may get this error if you are using SSL inspection in a proxy between your server and Cloudflare. To solve this:
+* Add the certificate to the system certificate pool.
+* Use the `--origin-ca-pool` flag and specify the path to the certificate.
+* Use the `--no-tls-verify` flag to stop `cloudflared` checking the certificate for a trust chain.
+
+## I see an error 1033 when attempting to run a tunnel.
+
+An error 1033 indicates your tunnel is not connected to Cloudflare's edge, and therefore cannot correctly route traffic to an origin. Make sure you completed step 5 in the [Tunnel guide](/connections/connect-apps/install-and-setup/tunnel-guide#5-start-routing-traffic) and assigned a CNAME record to points traffic to your tunnel. Alternatively, check [this guide](/connections/connect-apps/routing-to-tunnel/lb) to route traffic to your tunnel using load balancers.
```
Comment (sudarshan-reddy) [inline products/cloudflare-one/src/content/faq/teams-troubleshooting.md]: I vote we should simply redirect this here: https://support.cloudflare.com/hc/en-us/articles/360029779472-Troubleshooting-Cloudflare-1XXX-errors#h_W81O7hTPalZtYqNYkIHgH.

File: products/cloudflare-one/src/content/connections/connect-apps/configuration/configuration-file/ingress.md (hunk 1 above)
Comment (abracchi-tw) [reply products/cloudflare-one/src/content/connections/connect-apps/configuration/configuration-file/ingress.md:39]: hmmm I see. then what's the point of using wildcards in the config file? what are some common use cases?

Comment (sudarshan-reddy) [reply products/cloudflare-one/src/content/connections/connect-apps/configuration/configuration-file/ingress.md:39]: Looks like readability. In light of the wiki article I withdraw this comment.

File: products/cloudflare-one/src/content/faq/teams-troubleshooting.md [hunk 3]
```diff
@@ -71,6 +71,19 @@ This is due to a Google policy change requiring you to set your Google Admin con
...
+This means the origin is using a certificate that `cloudflared` does not trust. For example, you may get this error if you are using SSL inspection in a proxy between your server and Cloudflare. To solve this:
+* Add the certificate to the system certificate pool.
+* Use the `--origin-ca-pool` flag and specify the path to the certificate.
+* Use the `--no-tls-verify` flag to stop `cloudflared` checking the certificate for a trust chain.
+
+## I see an error 1033 when attempting to run a tunnel.
+
+An error 1033 indicates your tunnel is not connected to Cloudflare's edge, and therefore cannot correctly route traffic to an origin. Make sure you completed step 5 in the [Tunnel guide](/connections/connect-apps/install-and-setup/tunnel-guide#5-start-routing-traffic) and assigned a CNAME record to points traffic to your tunnel. Alternatively, check [this guide](/connections/connect-apps/routing-to-tunnel/lb) to route traffic to your tunnel using load balancers. 
+
+To see a list of active tunnels and their connections, run `cloudflared tunnel list` .
```
Comment (kodster28) [inline products/cloudflare-one/src/content/faq/teams-troubleshooting.md]:
```suggestion
To see a list of active tunnels and their connections, run `cloudflared tunnel list`.
```

**Review by kodster28 — APPROVED** [review]

**Pull request merged**